creator-node/src/app.js

```diff
@@ -50,12 +52,13 @@ app.use(errorHandler)
 
 const initializeApp = (port, serviceRegistry) => {
   const storagePath = DiskManager.getConfigStoragePath()
+
   // TODO: Can remove these when all routes
   // consume serviceRegistry
   app.set('ipfsAPI', serviceRegistry.ipfs)
   app.set('storagePath', storagePath)
   app.set('redisClient', serviceRegistry.redis)
-  app.set('audiusLibs', serviceRegistry.libs)
+  app.set('audiusLibs', serviceRegistry.audiusLibs)
```

Comment:
corresponding change to `serviceRegistry.js` change which renames this prop for consistency